server/lib/settings.ts

```diff
@@ -59,6 +59,8 @@ export interface MainSettings {
   hideAvailable: boolean;
   localLogin: boolean;
   trustProxy: boolean;
+  quotaQuantity?: number;
+  quotaPeriod?: number;
```

Comment:
Nitpicking here, but I don't think "quantity" is the right descriptor.  It kind of sounds like the number of quotas.  Maybe either `quotaLimit` or `quotaRequests`?  And `quotaDays` so that it's clear to anyone looking at the code that it's a number of days?